projects/plugins/protect/src/js/components/firewall-header/index.jsx

```diff
@@ -164,9 +172,14 @@ const FirewallHeader = ( {
 				<Col>
 					{ 'on' === status && (
 						<>
-							<Text className={ classnames( styles.status, styles.active ) } variant={ 'label' }>
-								{ __( 'Active', 'jetpack-protect' ) }
-							</Text>
+							<Status
+								status="active"
+								label={
+									standaloneMode
+										? __( 'Standalone mode', 'jetpack-protect' )
+										: __( 'Active', 'jetpack-protect', /* dummy arg to avoid bad minification */ 0 )
+								}
+							/>{ ' ' }
```

Comment:
This label is a perfect start, I think we can iterate on this to add more context or useful info (will circle back on this one).

Reply:
Perhaps "Standalone mode active", just to keep it clear?

Reply:
>Perhaps "Standalone mode active"

I originally started with that but it felt a tad too long (I suppose especially since I am used to just seeing `Active`, here's an example:
<img width="536" alt="Screen Shot 2024-01-30 at 14 27 07" src="https://github.com/Automattic/jetpack/assets/43220201/da40a00e-877e-43ba-84a9-ca869e7eb335">

I am open to the change though as it does avoid confusion - thoughts?